### ISSUE TITLE
-y option does not work

### ISSUE THREAD

**Issue by maxbane** [jkramer/shell-fm#85]
```
% bin/shell-fm -y http://localhost:9090
shell-fm: hash.c:54: unset: Assertion `hash->content != ((void *)0)' failed.
[1]    29802 abort      bin/shell-fm -y http://localhost:9090
```

And yet the proxy on port 9090 definitely works:

```
% echo $http_proxy
http://localhost:9090
% curl -s www.google.com | head -c 20
<!doctype html><html%   
```

**Comment by jkramer** [comment]
Does it work without the proxy? The error doesn't look like it's related to the proxy setting. I just tested it with a proxy on the same port and the same shell-fm parameter and it works fine.

**Comment by maxbane** [comment]
Yeah, at home it works fine without a proxy. At work, I get the above when trying to use the proxy, and I have no way of testing without the proxy (note that the proxy is NOT authenticating or anything fancy like that).

When you tested it, did you clear your local session/token and revoke the shell-fm application authorization on the last.fm side? I.e., try starting from a totally clean, never-before-authenticated state.
